projects/1.20-fabric/assets/ribbits/ribbits/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+ "itemGroup.ribbits.general": "呱呱",
+
+ "block.ribbits.brown_toadstool": "棕色伞菌块",
+ "block.ribbits.red_toadstool": "红色伞菌块",
+ "block.ribbits.toadstool_stem": "伞菌茎",

```

```suggestion
 "block.ribbits.toadstool_stem": "伞菌柄",
```
原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **06.专有名词** 一节。
你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

> 原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。 具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **06.专有名词** 一节。 你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。 诸如这类的问题请在您翻译的所有文件中全部查明并修改。

好的，我会修改